.eleventy.js

```diff
@@ -64,11 +64,6 @@ module.exports = (config) => {
     array && sortBy(array, sortByKeys)
   ))
 
-  config.addShortcode(
-    'renderMinified',
-    (filePath) => parse(nunjucksEnv.render(filePath, {})).removeWhitespace().toString(),
-  )
-
```

Comment:
This was only a problem with the examples were inlined (when JavaScript is disabled now), but it's still a problem at the moment:

![image](https://user-images.githubusercontent.com/66470099/220867822-70e7d766-fbe7-4079-ae47-cb5305a001e7.png)

I couldn't get `html-minifier-terser` working here before because it was async and async in Nunjucks is complicated...

Might need a bit more thought.